src/Microsoft.ML.Dnn/DnnUtils.cs

```diff
@@ -89,6 +90,24 @@ internal static Session LoadTFSession(IExceptionContext ectx, byte[] modelBytes,
             }
             return new Session(graph);
         }
+        internal static void MaybeDownloadFile(Uri address, string fileName)
+        {
+            using (WebClient client = new WebClient())
```

Comment:
See the remarks on https://docs.microsoft.com/en-us/dotnet/api/system.net.webclient

>  Important
> We don't recommend that you use the WebClient class for new development. Instead, use the System.Net.Http.HttpClient class.